css-floatgrid.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Floatgrid</title>
+    <style type="text/css">
+    @import "https://yarnpkg.com/en/package/normalize.css";
+    .grid {
+      background-color: #eee;
+      border:1px solid #000;
+      margin: auto;
+    }
+    .col {
+      float:left;
+      background-color: #eee;
+/*      padding-right: 10px;*/
+
+    }
+    .grid-clear {
+      clear:both;
+    }
+    .col-1-4 {
+      width: 25%;
+      background-color: lightblue;
+    }
+    .col-3-4 {
+      width: 75%;
+      background-color: lightyellow;
+    }
+    .col-1-2 {
+      width: 50%;
+      background-color: lightseagreen;
+    }
+    .col-1-5 {
+      width: 20%;
+      background-color: lightgoldenrodyellow;
+    }
+    .col-4-5 {
+      width: 80%;
+      background-color: lightpink;
+    }
+    .col-3-5 {
+      width: 60%;
+      background-color: lightsalmon;
+    }
+    .col-1-3 {
+      width: 33.33%;
+      background-color: lightcyan;
+    }
+    .col-2-3 {
+      width: 66.66%;
+      background-color: lightslategrey;
+    }
+    </style>
+  </head>
+  <body>
+  <p>1/4 3/4</p>
+  <div class="grid">
+    <div class="col col-1-4">25%</div>
+    <div class="col col-3-4">75%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>1/4  1/2 1/4</p>
+  <div class="grid">
+    <div class="col col-1-4">25%</div>
+    <div class="col col-1-2">50%</div>
+    <div class="col col-1-4">25%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>1/5 4/5</p>
+  <div class="grid">
+    <div class="col col-1-5">20%</div>
+    <div class="col col-4-5">80%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>1/5 3/5 1/5</p>
+  <div class="grid">
+    <div class="col col-1-5">20%</div>
+    <div class="col col-3-5">60%</div>
+    <div class="col col-1-5">20%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>1/3 2/3</p>
+  <div class="grid">
+    <div class="col col-1-3">33.33%</div>
+    <div class="col col-2-3">66.66%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>2/3 1/3</p>
+  <div class="grid">
+    <div class="col col-2-3">66.66%</div>
+    <div class="col col-1-3">33.33%</div>
+    <div class="clear"></div>
+  </div>
+
+  <p>1/4 3/4</p>
+  <div class="grid">
+    <div class="col col-1-4">25%</div>
+    <div class="col col-3-4">75%</div>
+    <div class="clear"></div>
+  </div>
+  </body>
```

Comment:
Don't forget to use your text editor's formatter before you commit. You have cosmetic indentation issues here.

Reply:
done